AI/VCAI/Goals/CollectRes.cpp

```diff
@@ -0,0 +1,208 @@
+/*
+ * Goals.cpp, part of VCMI engine
```

Comment:
Please fix all filenames in headers so we don't have to do it after.

Reply:
Ok, will do